app/view_utils/marketplace_helper.rb

```diff
@@ -0,0 +1,9 @@
+module MarketplaceHelper
+  module_function
+
+  def google_maps_key(cid)
+    community_key = Community.where(id: cid).pluck(:google_maps_key).first
```

Comment:
We need `where` here, as it returns an `ActiveRecord::Relation`, on which we can call `pluck` to avoid building a full ActiveRecord object for the community.